webapp/locales/de.json

```diff
@@ -562,8 +562,8 @@
       "hidden": "Geheim — Gruppe (auch der Name) komplett unsichtbar",
       "public": "Öffentlich — Gruppe und alle Beiträge für registrierte Nutzer sichtbar"
     },
-    "unmute": "Gruppe nicht mehr stummschalten",
-    "unmuted": "Gruppe nicht mehr stummgeschaltet",
+    "unmute": "Nicht stummschalten",
+    "unmuted": "Gruppe nicht stummgeschaltet",
```

Comment:
Does the success toast message also have issues with too long strings? If not, I would leave the one I put because it's a bit more meaningful.

Reply:
In the toast is no problem.

Done